### DISCUSSION TITLE
DC 1.1.12 how disable the fade effect when renaming file ?

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1575] [General]
Hello. Thanks for reading my topic.
In current version 1.1.12 when renaming an file happen an fade effect using an black color between the previous to renamed file naming.
Is possible disable it or change the color used in fade effect ?

Have an nice day.

**Comment by alexx2000** [comment]
Hi, see [Highlight new and updated files](https://doublecmd.github.io/doc/en/configuration.html#ConfigViewEx) option.

**Comment by Augusto7743** [comment]
Thanks for replying.
Now the fade effect is more long time and not is possible change the color.
